lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/Runner.scala

```diff
@@ -303,3 +313,9 @@ class Runner(
     }
   }
 }
+
+object Runner {
+
+  private val LAUNCHER_ENV_NAME = "ENSO_LAUNCHER"
```

Comment:
Compared to [2024.5.2 release](https://github.com/enso-org/enso/releases/tag/2024.5.2) there is new, user facing environment variable `ENSO_LAUNCHER` - e.g. new API. Should such environment variable be supported in the next official release, it has to be documented.

Reply:
In the most recent version of this PR, the `ENSO_LAUNCHER` environment variable is gone. Good.

Reply:
- #13225 attempts to
- remove support for `ENSO_LAUNCHER=shell`